datacapture/src/test/java/com/google/android/fhir/datacapture/QuestionnaireItemTranslationExtensionTest.kt

```
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.fhir.datacapture
+
+import android.os.Build
+import com.google.common.truth.Truth.assertThat
+import java.util.Locale
+import org.hl7.fhir.r4.model.Extension
+import org.hl7.fhir.r4.model.Questionnaire
+import org.hl7.fhir.r4.model.StringType
+import org.hl7.fhir.r4.utils.ToolingExtensions
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+
+@RunWith(RobolectricTestRunner::class)
+@Config(sdk = [Build.VERSION_CODES.P])
+class QuestionnaireItemTranslationExtensionTest {
+  @Test
+  fun localizedValues_default() {
+    val questionnaireItem =
+      Questionnaire.QuestionnaireItemComponent().apply {
+        text = "Patient Information"
+        prefix = "One"
+      }
+
+    assertThat(questionnaireItem.localizedText()).isEqualTo("Patient Information")
+    assertThat(questionnaireItem.localizedPrefix()).isEqualTo("One")
+  }
+
+  @Test
+  fun localizedValues_vietnameseTranslationAndDefaultLocale_shouldGetDefault() {
```

```suggestion
  fun localizedValues_vietnameseTranslation_usLocale_shouldGetUs() {
```

---

@jingtang10 We can change it to `localizedText_vietnameseTranslation_usLocale_shouldGetDefault` as `shouldGetUs` would be misleading because the returned value is the default value and not tied to any language translation extension. 
What do you think?

---

sounds good